traduzione cluster e nodi (#368)
cncf/glossary: content/it/nodes.md

---
title: Nodi
status: Completed
category: Concetto
---

## Cos'è

> Un nodo è un computer che lavora insieme ad altri computer, o nodi, per realizzare un compito comune. Prendi per esempio il tuo portatile, il modem e la stampante. Sono tutti collegati alla tua rete wireless che comunicano e collaborano, ognuno rappresenta un nodo. Nel [cloud computing](/cloud_computing/), un nodo può essere un computer fisico, un computer virtuale, chiamato [VM](/virtual_machine/) , o anche un [container](/container/).

Reviewer: ```suggestion
Un nodo è un computer che lavora insieme ad altri computer, o nodi, per realizzare un compito comune. Prendi per esempio il tuo portatile, il modem e la stampante. Sono tutti collegati alla tua rete wireless che comunicano e collaborano: ognuno rappresenta un nodo. Nel [cloud computing](/cloud_computing/), un nodo può essere un computer fisico, un computer virtuale (chiamato [VM](/virtual_machine/)) o anche un [container](/container/).
```

> ## Quale problema affronta

Reviewer 2: Da Style Guide abbiamo deciso di utilizzare il titolo.
Quali problematiche affronta

> Mentre un'applicazione potrebbe (e molte le fanno) girare su una singola macchina, ci sono alcuni rischi legati a questo. Vale a dire che il malfunzionamento del sistema sottostante interromperà l'esecuzione dell'applicazione. Per risolvere questo problema, gli sviluppatori hanno iniziato a creare [applicazioni distribuite](/distributed_apps/) dove ogni processo viene eseguito su un differente nodo. Perciò, i nodi che eseguono le applicazioni o processi come parte di un gruppo formano un [cluster](/cluster/), o gruppo di nodi che lavorano insieme per raggiungere un obiettivo comune.

Reviewer: ```suggestion
Un'applicazione potrebbe (e in molti casi è così) girare su una singola macchina, ma ci sono alcuni rischi legati a questo. Il malfunzionamento del sistema sottostante, ad esempio, potrebbe interrompere l'esecuzione dell'applicazione. Per risolvere questo problema, gli sviluppatori hanno iniziato a creare [applicazioni distribuite](/distributed_apps/) dove ogni processo viene eseguito su un nodo differente. Perciò, i nodi che eseguono le applicazioni o processi come parte di un gruppo formano un [cluster](/it/cluster/), o gruppo di nodi che lavorano insieme per raggiungere un obiettivo comune.
```

## In che modo aiuta

> Un nodo fornisce una precisa unità di calcolo (memoria, CPU, rete) che puoi assegnare ad un cluster. In una cloud native piattaforma o applicazione cloud native un nodo rappresenta una singola unità che può eseguire il processo. In teoria, i singoli nodi sono privi di differenze nel senso che qualsiasi nodo di un particolare tipo è praticamente indistinguibile da qualsiasi altro nodo dello stesso tipo.

Reviewer: ```suggestion
Un nodo fornisce una precisa unità di calcolo (memoria, CPU, rete) che può essere assegnata ad un cluster. In una cloud native piattaforma o applicazione cloud native un nodo rappresenta una singola unità che può eseguire il processo. In teoria, i singoli nodi sono privi di differenze, nel senso che qualsiasi nodo di un particolare tipo è praticamente indistinguibile da qualsiasi altro nodo dello stesso tipo.
```
